### PR TITLE
bot testing : 100% test coverage 

### PR DESCRIPTION
**Pull request by DhruvThakker**
#122 Coverage 100% for link_shortener

### CODE REVIEW

**Comment by showell** [comment]
This looks good...can you squash into a single commit with a nice commit message?

**Comment by showell** [comment]
@DhruvThakker The code here looks pretty good.  Can you change the commit messages so they have a verb in them?  An example would be:

~~~
github_detail_bot: Add tests for bad requests.

(This module now has 100% coverage.)
~~~

I'm kind of on the fence of having the helper method being added to `test_lib.py`.  I would prefer the tests to just directly import the method from the other module, or, if you really want the helper, split out a separate commit for that.

Thanks for working on this!

**Comment by showell** [comment]
This is getting real close!  Can you put the logging.exception back in the github bot?  You should be able to mock it in the tests.

File: zulip_bots/zulip_bots/bots/github_detail/github_detail.py [hunk 1]
```diff
@@ -52,7 +52,6 @@ def get_details_from_github(self, owner: str, repo: str, number: str) -> Union[N
             r = requests.get(
                 self.GITHUB_ISSUE_URL_TEMPLATE.format(owner=owner, repo=repo, id=number))
         except requests.exceptions.RequestException as e:
-            logging.exception(str(e))
```
Comment (DhruvThakker) [inline zulip_bots/zulip_bots/bots/github_detail/github_detail.py]: Add : logging.exception

**Comment by showell** [comment]
Merged!  Thanks @DhruvThakker!

**Pull request merged**